neuro_san/internals/graph/activations/branch_activation.py

```diff
@@ -121,27 +123,27 @@ async def build(self) -> str:
         :return: A string representing a List of messages produced during this process.
         """
 
-        assignments = self.get_assignments()
-        instructions = self.get_instructions()
+        assignments: str = self.get_assignments()
+        instructions: str = self.get_instructions()
 
-        uuid_str = str(uuid.uuid4())
-        component_name = self.get_name()
-        unique_name = f"{uuid_str}_{component_name}"
+        uuid_str: str = str(uuid.uuid4())
+        component_name: str = self.get_name()
+        unique_name: str = f"{uuid_str}_{component_name}"
         await self.create_resources(unique_name, instructions, None)
 
-        command = self.get_command()
-
         # If there is command, combine it with assignment to be used as HumanMessage.
+        command: str = self.get_command()
         if command:
             assignments = assignments + "\n" + command
+
         run: Run = await self.run_context.submit_message(assignments)
         run = await self.run_context.wait_on_run(run, self.journal)
 
-        messages = await self.run_context.get_response()
+        messages: List[BaseMessage] = await self.run_context.get_response()
 
         messages = await self.integrate_callable_response(run, messages)
 
-        response = IntraAgentMessageUtils.generate_response(messages)
+        response: str = IntraAgentMessageUtils.generate_response(messages)
```

Comment:
This is the one place that calls IntraAgentMessageUtils, the new devil to eliminate.